src/reducers/tasks.reducer.js

```diff
@@ -147,7 +147,7 @@ export default (state: State = initialState, action: Action = {}) => {
         // For the eject task, we simply want to delete this task altogether.
         // TODO: We should probably do this in `REFRESH_PROJECTS_FINISH`, which
```

Comment:
I think we can remove the todo comment as I think it's OK to leave it here.
It's probably better to have it in `COMPLETE_TASK` as we could refactor this later. Something like trigger a special action on task complete. Not sure about the API but it would go at this location and not during refresh projects.